docs/orchestrator/index.md

```diff
@@ -0,0 +1,41 @@
+# ODTP Orchestrator Overview
+
+The Orchestrator is the Application behind ODTP: it can be installed and run. 
+
+``` mermaid
+graph TD;
+    ODTPOrchestartor --> |defines and runs| Executions
+    ODTPOrchestartor --> |registers| ODTPComponents
+    ODTPOrchestartor --> |provides service to| Users
+    ODTPOrchestartor --> |registers| DigitalTwins
+    Users --> |own| DigitalTwins
+    DigitalTwins --> |are collections of| Executions
+    Executions --> |are pipelines of| ODTPComponents
```

Comment:
`are workflows`?

Reply:
I did not mention workflows as they are not yet implemented. Currently the execution is everything at once: a Workflow template, a workflow and an execution.